apps/ios_rpc/README.md

```diff
@@ -114,19 +57,198 @@ cmake ..
   -DCMAKE_OSX_ARCHITECTURES=arm64
   -DCMAKE_OSX_DEPLOYMENT_TARGET=14.0
   -DCMAKE_BUILD_WITH_INSTALL_NAME_DIR=ON
-  -DCMAKE_XCODE_ATTRIBUTE_DEVELOPMENT_TEAM=XXXXXXXXXX  # insert your Team ID
   -DUSE_IOS_RPC=ON  # to enable build iOS RPC application from TVM project tree
-cmake --build . --target custom_dso_loader ios_rpc  # Will use custom DSO loader by default
-# Resulting iOS RPC app bundle will be placed in:
-# apps/ios_rpc/ios_rpc/src/ios_rpc-build/Build/Products/[CONFIG]-iphoneos/tvmrpc.app
+  -DUSE_METAL=ON    # to enable Metal runtime
+
+cmake --build . --target custom_dso_loader tvm_runtime
+```
+
+### Building iOS TVM RPC application
+Before start, please run [init_proj.py](./init_proj.py) to update XCode developer metadata:
+```shell
+python3 init_proj.py --team_id XXXXXXXXXX --tvm_build_dir "/path/to/tvm/ios/build/folder"
+```
+You can get value of your `team_id` in the following ways:
+- **You have registered Apple Developer Profile**. In this case you developer
+  Team ID available at https://developer.apple.com/account/#/membership
+- You are using your local developer profile. In this case run the command above
+  with `XXXXXXXXXX` instead of Team ID. Then open `tvmrpc.xcodeproj` by using
```

Comment:
```suggestion
- You are using your local developer profile. In this case, leave `XXXXXXXXXX` in the command
  instead of substituting a Team ID. Then open `tvmrpc.xcodeproj` by using
```

Reply:
Done